sdk/README.md

```diff
@@ -33,13 +33,13 @@ a basic sequential pipeline.
 
 4. Compile the sample pipeline:
 
-    - `cd sdk/samples/sequential`  
-    - `dsl-compile-tekton --py ./sequential.py --output pipeline.yaml`
+    - `cd sdk/samples`  
+    - `dsl-compile-tekton --py ./parallel_join.py --output pipeline.yaml`
     
 5. Run the sample pipeline on a Tekton cluster:
 
     - `kubectl apply -f pipeline.yaml`
-    - `tkn pipeline start sequential-pipeline`
+    - `tkn pipeline start parallel-and-sequential-pipeline`
```

Comment:
`tkn pipeline start parallel-and-sequential-pipeline`
-->
`tkn pipeline start parallel-join-pipeline`
or `tkn pipeline start parallel-pipeline` ... to keep name consistent with the original [KFP example](https://github.com/kubeflow/pipelines/blob/e344b74dfd6bf316900ed808a363de30d953b9f2/samples/core/parallel_join/parallel_join.py#L42)
in conjunction with name change comment below

Reply:
The original pipeline name is called `download_and_join`. Do you want me to rename it to `download_and_join` for consistency?
https://github.com/kubeflow/pipelines/blob/e344b74dfd6bf316900ed808a363de30d953b9f2/samples/core/parallel_join/parallel_join.py#L45

Reply:
yup, good point Tommy

Reply:
well, the file name (`parallel_join.py`), the function name (`download_and_join`) and the generated pipeline name (`parallel-pipeline`) are all different

Reply:
Actually I redo the example based on the kfp parallel join. This should be more consistent now.